listenbrainz/db/dump_manager.py

```diff
@@ -44,6 +45,17 @@
 
 cli = click.Group()
 
+def send_dump_creation_notification(dump_name, dump_type):
+    if not current_app.config['TESTING']:
+        dump_link = 'http://ftp.musicbrainz.org/pub/musicbrainz/listenbrainz/{}/{}'.format(dump_type, dump_name)
+        send_mail(
+            subject="ListenBrainz dump created - {}".format(dump_name),
+            text=render_template('emails/data_dump_created_notification.txt', dump_name=dump_name, dump_link=dump_link),
+            recipients=['param@metabrainz.org'],
```

Comment:
I think this recipient should be a mailing list. `listenbrainz-observability@metabrainz.org`?